components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/tests/Engine/pn_2.5_legacy.json

```diff
@@ -1,15 +1,16 @@
 ﻿[
-  {
-    "EndpointUrl": "opc.tcp://opcplc:50000",
-    "UseSecurity": false,
-    "OpcAuthenticationMode": "UsernamePassword",
-    "OpcAuthenticationUsername": "username",
-    "OpcAuthenticationPassword": "password",
-    "DataSetWriterGroup": "DataSetWriterGroup0",
-    "DataSetWriterId": "DataSetWriterId0",
-    "DataSetPublishingInterval": 10000,
-    "NodeId": {
-      "Identifier": "nsu=http://microsoft.com/Opc/OpcPlc/;s=FastUInt1"
+    {
+        "EndpointUrl": "opc.tcp://opcplc:50000",
+        "UseSecurity": false,
+        "OpcAuthenticationMode": "UsernamePassword",
+        "OpcAuthenticationUsername": "username",
+        "OpcAuthenticationPassword": "password",
+        "DataSetWriterGroup": "DataSetWriterGroup0",
+        "DataSetWriterId": "DataSetWriterId0",
+        "DataSetPublishingInterval": 10000,
+        "Tag": "Tag0",
```

Comment:
`Tags` field was not present in `pn.json` for `2.5` version of OPC Publisher. Can we please add it in some other JSON file and keep this one as it was ?

Reply:
Neither did `DataSetWriterId`, `DataSetWriterGroup` and `DataSetPublishingInterval`. but fine I'll remove `Tag`